compare-images

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python
+
+import json
+import optparse
+import os
+import sys
+from tabulate import tabulate
```

Comment:
TBH, only being allowed to import modules is the thing I liked least about Google's style guide. Writing `tabulate.tabulate` and `pprint.pprint` ensaddenates me.

When looking at the call site, I very rarely care where a function comes from. When I do, it's easy enough to search for the module or use my editor's go-to-definition feature.